packages/SwingSet/src/kernel/dynamicVat.js

```diff
@@ -1,3 +1,4 @@
+import { makeMarshal } from '@agoric/marshal';
```

Comment:
Beware that the startup refactoring that's in flight renames and restructures this file.  This may complicate merging.